MHImageTabBar/AppDelegate.swift

```diff
@@ -7,30 +7,30 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+    private func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
```

Comment:
You should ask Xcode, I'm not sure why. According to Apple documentation this method is optional and Xcode suggests making it private to silence a warning.